File: server/test-server.js
const express = require('express');
const app = express();

app.use(express.json());

// Health check
app.get('/health', (req, res) => {
    res.json({ status: 'healthy', timestamp: new Date().toISOString() });
});

// Test BUZ route
app.get('/api/buz-test', (req, res) => {
    res.json({ message: 'BUZ test successful!' });
});

const PORT = process.env.PORT || 3002;
app.listen(PORT, () => {
    console.log(`Test server running on port ${PORT}`);
});

module.exports = app;

File: python-services/test_server.py
#!/usr/bin/env python3
"""
Simple test server to verify Flask setup
"""

from flask import Flask, jsonify
from flask_cors import CORS

app = Flask(__name__)
CORS(app)

@app.route('/health', methods=['GET'])
def health_check():
    """Health check endpoint"""
    return jsonify({
        "success": True,
        "status": "healthy",
        "message": "Test server is running"
    }), 200

@app.route('/test', methods=['GET'])
def test_endpoint():
    """Test endpoint"""
    return jsonify({
        "success": True,
        "message": "Test endpoint working"
    }), 200

if __name__ == '__main__':
    print("Starting test server on port 5004...")
    app.run(host='0.0.0.0', port=57079, debug=False)
